README.md

```diff
@@ -21,10 +23,10 @@ cp .env.template .env.local
 # Replace your keys in new .env.local file
 ```
 
-Next, add `http://localhost:3000/api/authenticate_magic_link` as a login and signup magic link URL to the dashboard. Stytch, for security purposes, verifies your magic link URLs before they are sent. You can set these magic link URLs for your project in the [Magic link URLs tab](https://stytch.com/dashboard/magic-link-urls).
+Next we'll configure the appropriate redirect URLs for your project, you'll set these magic link URLs for your project in the [Redirect URLs](https://stytch.com/dashboard/redirect-urls) section of your Dashboard. Add `http://localhost:3000/api/authenticate_magic_link` as both a login and sign-up redirect URL.
```

Comment:
Do you want to mention the Vercel redirect URL here in the event user tries to deploy to vercel?

Reply:
A follow-up PR will be opened to add directions for Vercel as well as a "Deploy with Vercel" button. Once this is in main that can be configured in Vercel and then added here.